test/search/test_organized_index.cpp

```diff
@@ -344,7 +359,8 @@ TEST (PCL, Organized_Neighbor_Search_Pointcloud_Neighbours_Within_Radius_Search)
 TEST (PCL, Organized_Neighbor_Search_Pointcloud_Neighbours_Within_Radius_Search_Benchmark_Test)
 {
   constexpr unsigned int test_runs = 10;
-  srand (time (NULL));
+  const unsigned int seed = time (nullptr);
+  srand (seed);
```

Comment:
Should we be printing this seed as well? Or is this seed unused?

Reply:
It is used for `rand()` in line 396, but there is no assert/expect in this "test", so a `SCOPED_TRACE` would not do anything (this is the test that could be converted to a benchmark some time). I only changed this to keep the calls to `srand` somewhat uniform.